release-notes/Code_Libraries/Skyline.DataMiner.DataSources.OpenConfig/Skyline.DataMiner.DataSources.OpenConfig.Gnmi_5.x.md

```diff
@@ -10,7 +10,7 @@ uid: Skyline.DataMiner.DataSources.OpenConfig.Gnmi_5.x
 
 The `DataMinerConnectorDataMapper` retains the last known value for each cell in memory. Previously, when a gNMI notification was received to delete a row, the corresponding cell values were not cleared from memory. This issue has now been resolved.
 
-#### Support for notification throttling in the DataMinerConnectorDataMapper [ID 41467]
+#### Support for notification throttling in the DataMinerConnectorDataMapper [ID 41467] [ID 41177]
```

Comment:
```suggestion
#### Support for notification throttling in the DataMinerConnectorDataMapper [ID 41177]
```
Good catch, RN 414467 is actually incorrect and can be removed.
RN 41467 has been introduced and documented in Skyline.DataMiner.DataSources.OpenConfig.Gnmi Range 6.x